src/scss/overrides/_variables.scss

```diff
@@ -1,6 +1,54 @@
-$font-family-sans-serif:     'Fira Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
-$font-family-serif:          'Fira Sans', Georgia, 'Times New Roman', Times, serif;
-$font-family-monospace:      'Fira Sans', Menlo, Monaco, Consolas, 'Courier New', monospace;
-$font-family-base:           'Fira Sans', $font-family-sans-serif;
+html {
+  font-weight: 300;
+}
+
+$list-item-vertical-margin: 0.25em;
+
+
+$font-family-sans-serif: 'Fira Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+$font-family-base: $font-family-sans-serif;
+$link-color: $developer-blue;
 
 $enable-flex: true; // Turn on Flexbox support
+$enable-transitions: true;
+
+$font-size-h1: 3.8rem;
+$font-size-h2: 3.5rem;
+$font-size-h3: 2.6rem;
+$font-size-h4: 1.8rem;
+$font-size-h5: 1.25rem;
+$font-size-h6: 1rem;
+
+$body-color: $charcoal;
+$headings-color: $charcoal;
+$headings-line-height: 1;
+$headings-font-weight: 400;
+$headings-margin-bottom: 1.25rem;
+
+$display1-size: 4.5rem;
+$display1-weight: 600;
+$display2-size: 2.6rem;
+$display2-weight: 200;
+
+$btn-border-radius: 0;
+$btn-border-radius-sm: 0;
+$btn-border-radius-lg: 0;
+
+$btn-font-weight: 300;
+$btn-primary-bg: $nightly-blue;
+$btn-primary-border: darken($btn-primary-bg, 10%);
+$btn-secondary-bg: $aurora-purple;
+$btn-secondary-border: darken($btn-secondary-bg, 10%);
+$btn-secondary-color: $white;
+$btn-success-bg: $gecko-green;
+$btn-success-border: darken($btn-success-bg, 10%);
+$btn-info-bg: $mobile-blue;
+$btn-info-border: darken($btn-info-bg, 10%);
+$btn-warning-bg: $market-orange;
+$btn-warning-border: darken($btn-warning-bg, 10%);
+$btn-danger-bg: $dino-red;
+$btn-danger-border: darken($btn-danger-bg, 10%);
+
+$list-group-bg: none;
```

Comment:
This is resulting in an invalid rule:

<img width="197" alt="screen shot 2016-10-20 at 10 05 15 am" src="https://cloud.githubusercontent.com/assets/206379/19569972/c5d88678-96ac-11e6-92eb-0f6189819cfa.png">